services/members/handler.js

```diff
@@ -95,7 +104,26 @@ export const get = async (event, ctx, callback) => {
 export const getAll = async (event, ctx, callback) => {
   try {
     // scan the table
-    const members = await db.scan(MEMBERS2024_TABLE);
+    const year = event.queryStringParameters.year;
+
+    // Get the table name based on the year from query param
+    let tableName;
+    switch (year) {
+      case "2021":
+        tableName = MEMBERSHIPS2021_TABLE;
```

Comment:
rlly good work, make sure to include the environment in the table query, so when we are in PROD it shows the prod tables, look at line 51 for an example

Reply:
@stevenh17